sep.pyx

```diff
@@ -838,6 +857,16 @@ def sum_circle(np.ndarray data not None, x, y, r,
     maskthresh : float, optional
         Threshold for a pixel to be masked. Default is ``0.0``.
 
+    seg : `numpy.ndarray`, optional
+        Segmentation image with dimensions of `data` and dtype `np.float32`.
+    
+    seg_id : array_like, optional
+        Array of segmentation ids that correspond to the dimensions of `x` 
+        and `y`.  If the values in seg_id are negative, then the mask is 
+        generated requiring pixels within the segment.  Otherwise, mask 
+        pixels in `seg` with nonzero values different from `seg_id` for a 
+        given object in the list.  Must be provided along with `seg`.
```

Comment:
I found this a bit hard to parse the first time I read it. How about
```
Array of segmentation ids  used to mask additional pixels in the image.
Dimensions correspond to the dimensions of ``x`` and ``y``. The behavior differs
depending on whether ``seg_id`` is negative or positive. If ``seg_id`` is positive,
all pixels belonging to other objects are masked. (Pixel ``j, i`` is masked if
``seg[j, i] != seg_id and seg[j, i] != 0``). If ``seg_id`` is negative, all pixels
other than those belonging to the object of interest are masked. (Pixel ``j, i`` is
masked if ``seg[j, i] != -seg_id``).
```
Note the double backticks so that this renders correctly in restructured text docs (rather than the markdown single-backticks).

Reply:
Agreed.  Docstrings updated in 71328de0b2bf590b2fa0f5a62eb15095b89863a6.